lib/multi_patient_api/bulk_data_utils.rb

```diff
@@ -0,0 +1,383 @@
+require 'pry'
+require 'json/jwt'
+module BulkDataUtils
+
+	include Inferno::DSL::Assertions
```

Comment:
This shouldn't be necessary since these methods will be used in tests which already include the assertions.